processmanager/manager.go

```diff
@@ -294,33 +293,6 @@ func (pm *ProcessManager) ConvertTrace(trace *host.Trace) (newTrace *libpf.Trace
 	return newTrace
 }
 
-// findMappingForTrace locates the mapping for a given host trace.
```

Comment:
Moved without changes to `processinfo.go` for [consistency](https://github.com/open-telemetry/opentelemetry-ebpf-profiler/blob/84c6ca880ad4be106a760efb5ab555709f7cce8e/processmanager/processinfo.go#L6-L10).